systemd-udev-settle.service is deprecated

**Describe the bug**
upon running systemctl status systemd-udev-settle.service
the following logs are shown:

```
systemd[1]: Starting systemd-udev-settle.service - Wait for udev To Complete Device Initialization...
udevadm[854]: systemd-udev-settle.service is deprecated. Please fix cardwired.service not to pull it in.
systemd[1]: Finished systemd-udev-settle.service - Wait for udev To Complete Device Initialization.
```

**To Reproduce**
install cardwire
systemctl enable cardwired
reboot
systemctl status systemd-udev-settle.service

**Expected behavior**
systemd-udev-settle.service is not pulled in.

**Environment details**
Please provide your distro, kernel, and hardware details. The output of `fastfetch` (or `neofetch`) is preferred:
```text
luanv@tomahawk
--------------
OS: Fedora Linux 44 (KDE Plasma Desktop Edition) x86_64
Host: MS-7E59 (2.0)
Kernel: Linux 7.1.8-200.fc44.x86_64
Uptime: 20 mins
Packages: 5 (appimage), 34 (flatpak), 3672 (rpm)
Shell: zsh 5.9
Display (M28U): 3840x2160 @ 1.75x in 31", 144 Hz [External, HDR] *
Display (24G2W1G3-): 1920x1080 in 24", 144 Hz [External]
DE: KDE Plasma 6.7.4
WM: KWin (Wayland)
WM Theme: Oxygen
Theme: Oxygen (Dark) [Qt], Breeze-Dark [GTK2], Breeze [GTK3/4]
Icons: oxygen [Qt], oxygen [GTK2/3/4]
Font: Noto Sans (10pt) [Qt], Noto Sans (10pt) [GTK2/3/4]
Cursor: Oxygen_Zion (24px)
Terminal: konsole 26.4.3
Terminal Font: Cascadia Code NF (10pt)
CPU: AMD Ryzen 7 9800X3D (16) @ 5.27 GHz
GPU 1: AMD Radeon Graphics [Integrated]
GPU 2: NVIDIA GeForce RTX 4070 Ti SUPER [Discrete]
Memory: 9.26 GiB / 30.25 GiB (31%)
Swap: 0 B / 35.00 GiB (0%)
Disk (/): 311.04 GiB / 930.91 GiB (33%) - btrfs
Disk (/mnt/data): 854.99 GiB / 1.82 TiB (46%) - btrfs
Local IP (enp7s0): 192.168.50.20/24
Locale: pt_BR.UTF-8

```
Cardwire version (`cardwire --version`): cardwire-cli 0.12.1

**`cardwire list` output**
Please paste the text output (do not use a screenshot):
```text
ID  NAME                                  PCI           RENDER      CARD   DEFAULT DISCRETE  BLOCKED
--  ------------------------------------  ------------  ----------  -----  -------  -------  -------
0   NVIDIA GeForce RTX 4070 Ti SUPER      0000:01:00.0  renderD128  card1  (*)      (*)      false
1   AMD Ryzen 7 9800X3D 8-Core Processor  0000:79:00.0  renderD129  card2  ( )      ( )      false
```

**`ls -la /dev/dri/by-path/`**
```text
drwxr-xr-x root root 140 B ter 18 ago 2026 16:25:17 -03:00  .
drwxr-xr-x root root 140 B ter 18 ago 2026 16:25:17 -03:00  ..
lrwxrwxrwx root root   8 B ter 18 ago 2026 16:25:17 -03:00  pci-0000:01:00.0-card ➜ ../card1
lrwxrwxrwx root root   8 B ter 18 ago 2026 16:25:11 -03:00  pci-0000:01:00.0-platform-simple-framebuffer.0-card ➜ ../card0
lrwxrwxrwx root root  13 B ter 18 ago 2026 16:25:17 -03:00  pci-0000:01:00.0-render ➜ ../renderD128
lrwxrwxrwx root root   8 B ter 18 ago 2026 16:25:17 -03:00  pci-0000:79:00.0-card ➜ ../card2
lrwxrwxrwx root root  13 B ter 18 ago 2026 16:25:17 -03:00  pci-0000:79:00.0-render ➜ ../renderD129
```

**cardwired logs**
Please provide the daemon logs (e.g., run `journalctl -eu cardwired.service --no-pager`):
```text
ago 18 16:25:21 tomahawk systemd[1]: Starting cardwired.service - Cardwire Daemon...
ago 18 16:25:21 tomahawk (cardwired)[1161]: cardwired.service: ConfigurationDirectory 'cardwire' already exists but the mode is different. (File system: 755 ConfigurationDirectoryMode: 700)
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] IOMMU detected, reading pci devices using iommu dir
ago 18 16:25:21 tomahawk cardwired[1161]: [WARN ] llvmpipe (LLVM 22.1.8, 256 bits): Not available (VK_EXT_pci_bus_info not supported)
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] Successfully resolved card1 and renderD128 for PCI 0000:01:00.0
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] Successfully resolved card2 and renderD129 for PCI 0000:79:00.0
ago 18 16:25:21 tomahawk cardwired[1161]: [WARN ] AMD Ryzen 7 9800X3D 8-Core Processor: EGL discrete check failed: EGL driver found the device but doesn't support querying GPU type
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] gpu NVIDIA GeForce RTX 4070 Ti SUPER id: 0 internal: 0, desktop: 4, connected: 2, total: 4, connected_internal: 0, connected_desktop: 2
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] gpu AMD Ryzen 7 9800X3D 8-Core Processor id: 1 internal: 0, desktop: 5, connected: 0, total: 5, connected_internal: 0, connected_desktop: 0
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] inserting: 1161 into map
ago 18 16:25:21 tomahawk cardwired[1161]: [ERROR] (ignoring) failed to get inode for hwmon 0000:01:00.0: No such file or directory (os error 2)
ago 18 16:25:21 tomahawk cardwired[1161]: [WARN ] failed to get inode for /sys/class/backlight/nvidia_0: No such file or directory (os error 2)
ago 18 16:25:21 tomahawk cardwired[1161]: [ERROR] (ignoring) failed to get inode for backlight nvidia_0: No such file or directory (os error 2)
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] Switched to Manual
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] write_command; command=Auth(Some(External), Some([48]))
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] write_commands; commands=[Auth(Some(External), Some([48]))] extra_bytes=None
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] read_command;
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] read_commands; n_commands=1
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] write_commands; commands=[NegotiateUnixFD, Begin] extra_bytes=Some([108, 1, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 109, 0, 0, 0, 1, 1, 111, 0, 21, 0, 0, 0, 47, 111, 114, 103, 47, 102, 114, 101, 101, 100, 101, 115, 107, 116, 111, 112, 47, 68, 66, 117, 115, 0, 0, 0, 2, 1, 115, 0, 20, 0, 0, 0, 111, 114, 103, 46, 102, 114, 101, 101, 100, 101, 115, 107, 116, 111, 112, 46, 68, 66, 117, 115, 0, 0, 0, 0, 3, 1, 115, 0, 5, 0, 0, 0, 72, 101, 108, 108, 111, 0, 0, 0, 6, 1, 115, 0, 20, 0, 0, 0, 111, 114, 103, 46, 102, 114, 101, 101, 100, 101, 115, 107, 116, 111, 112, 46, 68, 66, 117, 115, 0, 0, 0, 0])
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] read_commands; n_commands=1
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] start_object_server; started_event=Some(Event { listeners_notified: 0, listeners_total: 1 })
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] obj_server_task;
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] monitor_name_lost; name=org.opengamingcollective.cardwire
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] write_command; command=Auth(Some(External), Some([48]))
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] write_commands; commands=[Auth(Some(External), Some([48]))] extra_bytes=None
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] read_command;
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] read_commands; n_commands=1
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] write_commands; commands=[NegotiateUnixFD, Begin] extra_bytes=Some([108, 1, 0, 1, 0, 0, 0, 0, 5, 0, 0, 0, 109, 0, 0, 0, 1, 1, 111, 0, 21, 0, 0, 0, 47, 111, 114, 103, 47, 102, 114, 101, 101, 100, 101, 115, 107, 116, 111, 112, 47, 68, 66, 117, 115, 0, 0, 0, 2, 1, 115, 0, 20, 0, 0, 0, 111, 114, 103, 46, 102, 114, 101, 101, 100, 101, 115, 107, 116, 111, 112, 46, 68, 66, 117, 115, 0, 0, 0, 0, 3, 1, 115, 0, 5, 0, 0, 0, 72, 101, 108, 108, 111, 0, 0, 0, 6, 1, 115, 0, 20, 0, 0, 0, 111, 114, 103, 46, 102, 114, 101, 101, 100, 101, 115, 107, 116, 111, 112, 46, 68, 66, 117, 115, 0, 0, 0, 0])
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] read_commands; n_commands=1
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] start_object_server; started_event=Some(Event { listeners_notified: 0, listeners_total: 1 })
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] obj_server_task;
ago 18 16:25:21 tomahawk systemd[1]: Started cardwired.service - Cardwire Daemon.
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] monitor_name_lost; name=net.hadess.SwitcherooControl
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] Switcheroo DBus shim started successfully
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] Daemon started succesfully
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] write_command; command=Auth(Some(External), Some([48]))
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] write_commands; commands=[Auth(Some(External), Some([48]))] extra_bytes=None
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] read_command;
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] read_commands; n_commands=1
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] write_commands; commands=[NegotiateUnixFD, Begin] extra_bytes=Some([108, 1, 0, 1, 0, 0, 0, 0, 11, 0, 0, 0, 109, 0, 0, 0, 1, 1, 111, 0, 21, 0, 0, 0, 47, 111, 114, 103, 47, 102, 114, 101, 101, 100, 101, 115, 107, 116, 111, 112, 47, 68, 66, 117, 115, 0, 0, 0, 2, 1, 115, 0, 20, 0, 0, 0, 111, 114, 103, 46, 102, 114, 101, 101, 100, 101, 115, 107, 116, 111, 112, 46, 68, 66, 117, 115, 0, 0, 0, 0, 3, 1, 115, 0, 5, 0, 0, 0, 72, 101, 108, 108, 111, 0, 0, 0, 6, 1, 115, 0, 20, 0, 0, 0, 111, 114, 103, 46, 102, 114, 101, 101, 100, 101, 115, 107, 116, 111, 112, 46, 68, 66, 117, 115, 0, 0, 0, 0])
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] read_commands; n_commands=1
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] {}; task_name="org.freedesktop.UPower proxy caching"
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] successfully sent reset-failed on nvidia-powerd.service
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] successfully sent start on nvidia-powerd.service
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] NVIDIA GeForce RTX 4070 Ti SUPER: Power state changed: D0
ago 18 16:25:21 tomahawk cardwired[1161]: [INFO ] AMD Ryzen 7 9800X3D 8-Core Processor: Power state changed: D0
ago 18 16:46:49 tomahawk cardwired[1161]: [INFO ] dispatch_call; msg=Msg { type: MethodCall, serial: 2, sender: UniqueName(":1.289"), path: ObjectPath("/org/opengamingcollective/cardwire"), iface: InterfaceName("org.opengamingcollective.cardwire.Manager"), member: MemberName("Status"), fds: [] } hdr=Header { primary: PrimaryHeader { endian_sig: Little, msg_type: MethodCall, flags: BitFlags<Flags>(0b0), protocol_version: 1, body_len: 0, serial_num: 2 }, fields: Fields { path: Some(ObjectPath("/org/opengamingcollective/cardwire")), interface: Some(InterfaceName("org.opengamingcollective.cardwire.Manager")), member: Some(MemberName("Status")), error_name: None, reply_serial: None, destination: Some(BusName::WellKnown("org.opengamingcollective.cardwire")), sender: Some(UniqueName(":1.289")), signature: Unit, unix_fds: None } }
ago 18 16:46:49 tomahawk cardwired[1161]: [INFO ] dispatch_call; msg=Msg { type: MethodCall, serial: 3, sender: UniqueName(":1.289"), path: ObjectPath("/org/opengamingcollective/cardwire"), iface: InterfaceName("org.freedesktop.DBus.ObjectManager"), member: MemberName("GetManagedObjects"), fds: [] } hdr=Header { primary: PrimaryHeader { endian_sig: Little, msg_type: MethodCall, flags: BitFlags<Flags>(0b0), protocol_version: 1, body_len: 0, serial_num: 3 }, fields: Fields { path: Some(ObjectPath("/org/opengamingcollective/cardwire")), interface: Some(InterfaceName("org.freedesktop.DBus.ObjectManager")), member: Some(MemberName("GetManagedObjects")), error_name: None, reply_serial: None, destination: Some(BusName::WellKnown("org.opengamingcollective.cardwire")), sender: Some(UniqueName(":1.289")), signature: Unit, unix_fds: None } }
ago 18 16:46:49 tomahawk cardwired[1161]: [INFO ] dispatch_call; msg=Msg { type: MethodCall, serial: 4, sender: UniqueName(":1.289"), path: ObjectPath("/org/opengamingcollective/cardwire/Gpu/0"), iface: InterfaceName("org.opengamingcollective.cardwire.Gpu"), member: MemberName("GetDevice"), fds: [] } hdr=Header { primary: PrimaryHeader { endian_sig: Little, msg_type: MethodCall, flags: BitFlags<Flags>(0b0), protocol_version: 1, body_len: 0, serial_num: 4 }, fields: Fields { path: Some(ObjectPath("/org/opengamingcollective/cardwire/Gpu/0")), interface: Some(InterfaceName("org.opengamingcollective.cardwire.Gpu")), member: Some(MemberName("GetDevice")), error_name: None, reply_serial: None, destination: Some(BusName::WellKnown("org.opengamingcollective.cardwire")), sender: Some(UniqueName(":1.289")), signature: Unit, unix_fds: None } }
ago 18 16:46:49 tomahawk cardwired[1161]: [INFO ] dispatch_call; msg=Msg { type: MethodCall, serial: 5, sender: UniqueName(":1.289"), path: ObjectPath("/org/opengamingcollective/cardwire/Gpu/1"), iface: InterfaceName("org.opengamingcollective.cardwire.Gpu"), member: MemberName("GetDevice"), fds: [] } hdr=Header { primary: PrimaryHeader { endian_sig: Little, msg_type: MethodCall, flags: BitFlags<Flags>(0b0), protocol_version: 1, body_len: 0, serial_num: 5 }, fields: Fields { path: Some(ObjectPath("/org/opengamingcollective/cardwire/Gpu/1")), interface: Some(InterfaceName("org.opengamingcollective.cardwire.Gpu")), member: Some(MemberName("GetDevice")), error_name: None, reply_serial: None, destination: Some(BusName::WellKnown("org.opengamingcollective.cardwire")), sender: Some(UniqueName(":1.289")), signature: Unit, unix_fds: None } }
```

**Additional context**
looking at the output of systemd-analize plot, systemd-udev-settle.service takes 6 seconds to start

--- a/assets/cardwired.service
+++ b/assets/cardwired.service
@@ -1,8 +1,6 @@
 [Unit]
 Description=Cardwire Daemon
 After=dbus.service
-Wants=systemd-udev-settle.service
-After=systemd-udev-settle.service
 
 
 [Service]

